cocos/3d/skinned-mesh-renderer/skinned-mesh-batch-renderer.ts

```diff
@@ -469,7 +469,7 @@ export class SkinnedMeshBatchRenderer extends SkinnedMeshRenderer {
                 region.texExtent.height = unit.size.y * this.atlasSize;
                 const { data } = partial.image;
                 if (!ArrayBuffer.isView(data)) {
-                    texImages.push(data);
+                    texImages.push(data as TexImageSource);
```

Comment:
does `pal/image` module provides a type called `TexImageSource` ? 

It's currently defined in lib.dom.d.ts

Reply:
'SourceImage' is partially compatible with 'TexImageSource'. If you don't add 'as' it will report an error.